Ben quiz

benwilburn opened the pull request:
This is my quiz submission. I feel like something is missing but I can't see it.

benwilburn committed:
created file

benwilburn committed:
added the basic structure for the page

benwilburn committed:
added css styling & images

benwilburn committed:
commiting index.html again

sscotth commented on index.html:
```diff
@@ -0,0 +1,254 @@
+		</article>
+	</div>
+</body>
+</html>
```
Always include a new line at the end of every file. Consider adding ensure_newline_at_eof_on_save to your sublime preferences: http://docs.sublimetext.info/en/latest/reference/settings.html#automatic-behavior

sscotth commented on main.css:
```diff
@@ -0,0 +1,90 @@
+.availabilitydiv p {
+	color: yellow;
+	font-size: 14px;
+}
```
Always include a new line at the end of every file. Consider adding ensure_newline_at_eof_on_save to your sublime preferences: http://docs.sublimetext.info/en/latest/reference/settings.html#automatic-behavior

sscotth commented on index.html:
```diff
@@ -0,0 +1,254 @@
+<!-- QUIZ TIME
```
Never put a comment before the doctype. The browser needs to know it's dealing with HTML before anything else.

sscotth commented on index.html:
```diff
@@ -0,0 +1,254 @@
+
+
+
+-->
```
Large comments like this are probably best staying in the README, but if you insist on having them in the HTML, I would place them at the bottom. Your browser has to download and parse all the text above before it can begin rendering. Also it fails validation: Error: A “charset” attribute on a “meta” element found after the first 1024 bytes.

benwilburn replied:
Meant to remove when I was done with the exercise

sscotth commented on index.html:
```diff
@@ -0,0 +1,254 @@
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title></title>
```
Error: Element “title” must not be empty.

sscotth commented on index.html:
```diff
@@ -0,0 +1,254 @@
+<body>
+	<div class="site">
+		<article class="card">
+			<header class="headingdiv">
```
You do not need a class on the `header` tag if all the `header`s have the same class. Just use the css of 

``` css
header {

}
```

instead of

``` css
.headingdiv {

}
```

sscotth commented on index.html:
```diff
@@ -0,0 +1,254 @@
+				<div class="imgdivpro imgdiv">
+					<img src="images/macpro2.jpeg" alt="MacPro">
+				</div>
+				<div class="descriptiondiv">
```
I would seriously consider avoiding naming conventions like this. First, creating a class that has the tag name in it is superfluous. Just use `<div class="description">`

Second, if you insist on targeting a `class="description"` that is specifically a `div` then you can use

``` css
div.description {

}
```

sscotth commented on index.html:
```diff
@@ -0,0 +1,254 @@
+				</div>
+				<div>
+					<p>10 Units: $2,399.00 ea</p>
+				</div>
```
Why are you putting `div`s around individual `p` tags?

benwilburn replied:
meant to take those out

sscotth commented on main.css:
```diff
@@ -0,0 +1,90 @@
+	margin: 0;
+}
+
+.specificationsection h3 {
```
heading tags should not be qualified. simply use:

``` css
h3 {

}
```

> "Heading elements (h1-h6) should be defined as top-level styles and not scoped to particular areas of the page. The headings are considered to be built-in objects in Object-Oriented CSS, and their appearance should be consistent across an entire site. This allows those styles to be reused across your site for better visual consistency and performance and easier maintenance.

sscotth commented on main.css:
```diff
@@ -0,0 +1,90 @@
+
+.pricingsection h3 {
+	margin: 0;
+}
```
When you have duplicate styles like the above try using: 

``` css
.pricingsection div p, .pricingsection h3 {
        margin: 0;
}
```

benwilburn committed:
made suggested changes

benwilburn committed:
made other small changes

benwilburn committed:
trying to fix merge conflict

benwilburn committed:
Merge branch 'master' into benQuiz

benwilburn commented:
@sscotth

benwilburn committed:
added a line to the end of both pages

sscotth commented on main.css:
```diff
@@ -0,0 +1,86 @@
+
+.pricing div p, {
+	margin: 0;
+}
```
This is invalid. Accidental comma?

sscotth commented:
Remove the `.DS_Store` file

sscotth commented:
At lower resolutions, your images overlap and your cards are no longer 3 across. Can you fix?
<img width="461" alt="screen shot 2016-04-19 at 10 25 11" src="https://cloud.githubusercontent.com/assets/1010488/14644941/62cf9e64-0619-11e6-831a-42637b3a90f4.png">
<img width="455" alt="screen shot 2016-04-19 at 10 25 19" src="https://cloud.githubusercontent.com/assets/1010488/14644942/62d136ca-0619-11e6-916c-475a91ff89ea.png">

benwilburn committed:
made the recommended changes

I also fixed the images to be responsive and got the cards to line up correctly due to vertical-align: top

benwilburn commented:
@sscotth

sscotth commented:
👍

benwilburn commented:
@sscotth am I suppose to merge to master or leave it?
